main.py

```diff
@@ -611,47 +611,47 @@ def get(self):
+        verification = self.get_db()
 
         html = "<h1>Stats</h1>"
 
         html += "<h2>" + str(len(verification))
         html += " unique emails entered</h2>"
 
-        only_verified = sorted([ k for k,v in verification.iteritems() if v['verified'] == 1 and v['unverified'] == 0 ])
+        only_verified = sorted([ k for k,v in verification.iteritems() if len(v['verified']) == 1 and len(v['unverified']) == 0 ])
         html += "<h3>" + str(len(only_verified)) + " emails in good condition</h3>"
         for email in only_verified:
             html += email + "<br>"
 
-        verified_and_unverified = sorted([ k for k,v in verification.iteritems() if v['verified'] >= 1 and v['unverified'] >= 1 ])
+        verified_and_unverified = sorted([ k for k,v in verification.iteritems() if len(v['verified']) >= 1 and len(v['unverified']) >= 1 ])
         html += "<h3>" + str(len(verified_and_unverified)) + " emails with verified and unverified records</h3>"
         for email in verified_and_unverified:
-            html += email + " [" + str(verification[email]['unverified']) + "]<br>"
+            html += email + " [" + str(len(verification[email]['unverified'])) + "]<br>"
 
-        only_unverified = sorted([ k for k,v in verification.iteritems() if v['verified'] == 0 and v['unverified'] >= 1 ])
+        only_unverified = sorted([ k for k,v in verification.iteritems() if len(v['verified']) == 0 and len(v['unverified']) >= 1 ])
         html += "<h3>" + str(len(only_unverified)) + " emails with only unverified records</h3>"
         for email in only_unverified:
-            html += email + " [" + str(verification[email]['unverified']) + "]<br>"
+            html += email + " [" + str(len(verification[email]['unverified'])) + "]<br>"
 
-        multiple_verified = sorted([ k for k,v in verification.iteritems() if v['verified'] > 1])
+        multiple_verified = sorted([ k for k,v in verification.iteritems() if len(v['verified']) > 1 ])
         # If there are ever any entries in multiple_verified, the DB is in a very bad state
         if multiple_verified:
-            html += "<h3>Attention! There are " + str(len(only_unverified)) + " emails with more than one verified record. The DB is REALLY messed up!</h3>"
+            html += "<h3>Attention! There are " + str(len(multiple_verified)) + " emails with more than one verified record. The DB is REALLY messed up!</h3>"
             for email in multiple_verified:
-                html += email + " [" + str(verification[email]['verified']) + ", " + str(verification[email]['unverified']) + "]<br>"
+                html += email + " [" + str(len(verification[email]['verified'])) + ", " + str(len(verification[email]['unverified'])) + "]<br>"
 
         html += """
         <script>
           function sendEmails(action) {
+            window.alert("Cleaning up DB, press OK to continue");
             var data = new FormData();
             data.append('action', action);
```

Comment:
I think it would be better to have this as a URL path param, e.g. /admin/cleanup, /admin/emailblast